docs/config.rst

```diff
@@ -194,6 +194,15 @@ Features
 
    See also: :attr:`LP_MARK_DISABLED`.
 
+.. attribute:: LP_ENABLE_AWS_PROFILE
+   :type: bool
+   :value: 1
+
+   Display the current value of the ``AWS_PROFILE`` or ``AWS_DEFAULT_PROFILE``
+   environment variables if they are set.
```

Comment:
This would be the place to add link(s) to the tools that this is displaying.

I'm assuming this is for the AWS CLI, so maybe his link? https://docs.aws.amazon.com/cli/latest/userguide/cli-configure-envvars.html
```suggestion
   environment variables if they are set. These variables are used to
   configure the `AWS CLI`_.
   
   .. _`AWS CLI`: https://docs.aws.amazon.com/cli/latest/userguide/cli-configure-envvars.html
```

Reply:
I've added a slightly different link that lands on the AWS documentation about named profiles, this seems more targetted.